cortex-m/Cargo.toml

```diff
@@ -34,6 +34,7 @@ cm7-r0p1 = ["cm7"]
 linker-plugin-lto = []
 std = []
 critical-section-single-core = ["critical-section/restore-state-bool"]
+coprocessor = []
```

Comment:
Yes, if people use these functions in targets without coprocessors, the target can either stall or crash (depends on the target it's a HardFault or UsageFault)

Reply:
Sure, but it being behind a feature doesn't affect that really, someone could still enable the feature without a coprocessor. It would be helpful to mention this in the function documentation, but I don't think there's any reason to feature-gate it.

Reply:
Feature gate is removed